layouts/_default/taxonomy.html

```diff
@@ -29,7 +29,7 @@ <h3><a href="{{ $manualLink }}"{{ with .Params.manualLinkTitle }} title="{{ . }}
       {{ end }}
     </div>
     <div class="mt-5 mb-5">
-    <a href="{{ with .Site.GetPage (printf "/%s" ( lower .Data.Plural )) }}{{ .RelPermalink }}{{ end }}" class="all-tax-terms">{{ humanize ( T "ui_see_all" ) }} {{ with .Data.Plural }}{{ . | humanize }}{{ end }}</a>
+    <a href="{{ with .Site.GetPage (printf "/%s" ( lower .Data.Plural )) }}{{ .RelPermalink }}{{ end }}" class="all-tax-terms">{{ humanize ( T "ui_see_all" ) }} {{ humanize ( T "ui_tags" ) }}</a>
```

Comment:
```suggestion
    <a href="{{ with .Site.GetPage (printf "/%s" ( lower .Data.Plural )) }}{{ .RelPermalink }}{{ end }}" class="all-tax-terms">{{ humanize ( T "ui_see_all" ) }}</a>
```
Some languages have different word order with English. So, this will make localization difficult.

How about removing the ordered string pattern.

[ui_see_all]
other = "See all"

--> 

[ui_see_all_tags]
other = "See all Tags"

Reply:
ok, please see my latest commit